service/src/java/org/apache/hive/service/cli/thrift/ThriftCLIService.java

```diff
@@ -536,7 +538,7 @@ public TCloseSessionResp CloseSession(TCloseSessionReq req) throws TException {
         context.setSessionHandle(null);
       }
     } catch (Exception e) {
-      LOG.warn("Error closing session: ", e);
+      LOG.warn("Error closing session", e);
```

Comment:
should this be error level log? or maybe change it to "Failed to close the session"